orc8r/cloud/go/services/orchestrator/servicers/builder_servicer.go

```diff
@@ -191,3 +198,17 @@ func getVpnMconfig(gatewayConfig *models.MagmadGatewayConfigs) *mconfig_protos.O
 
 	return ret
 }
+
+func getStateMconfig(net configurator.Network, gwKey string) *mconfig_protos.State {
+	mconfigProto := &mconfig_protos.State{
+		SyncInterval: 60,
+		LogLevel:     protos.LogLevel_INFO,
+	}
+	netConfig := net.Configs["state_config"]
+	if netConfig != nil {
+		nsConfig := netConfig.(*models.StateConfig)
+		syncInterval := nsConfig.SyncInterval
+		mconfigProto.SyncInterval = math.JitterUint32(syncInterval, gwKey, 0.25)
+	}
+	return mconfigProto
```

Comment:
Not quite right -- we want to jitter the value regardless of whether we use the default proto or augment it from the netconfig

Also add a regression test for this affordance